robot/basestation/static/css/rover.css

```diff
@@ -198,11 +198,7 @@ textarea#rover-log {
 .imageWrapper {
   position: relative;
 }
-.overlayImage {
```

Comment:
@vashmata It was or the GPS map placeholder image. Overlay is not needed anymore. The person probably used overlay because he was unable to place it properly. The relative position works fine now

Reply:
git blame to the rescue?
`744fb587 robot/basestation/static/css/rover.css (Josh Glazer   2019-06-30 02:59:14 -0400 201) .overlayImage {`

Reply:
Oh, this was used in the example code I found for placing dots as an overlay on the map. Maybe  in the example the overlay thing was actually used later